xbmc/cores/VideoPlayer/test/edl/TestEdl.cpp

```diff
@@ -325,17 +343,17 @@ TEST_F(TestEdl, TestCommBreakAdvancedSettings)
   EXPECT_EQ(found, true);
   // confirm the start and end times of all the commbreaks match
   EXPECT_EQ(edl.GetEditList().size(), 5);
-  EXPECT_EQ(edl.GetEditList().at(0).start, 10 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(0).end, 22 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(1).start, 30 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(1).end, 32 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(2).start, 37 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(2).end, 50 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(3).start, 52 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(3).end, 60 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(4).start, 62 * 1000);
-  EXPECT_EQ(edl.GetEditList().at(4).end, std::lround(65.1 * 1000));
-  // now lets change autowait and autowind and check the edits are correcly adjusted
+  EXPECT_EQ(edl.GetEditList().at(0).start, 10s);
+  EXPECT_EQ(edl.GetEditList().at(0).end, 22s);
+  EXPECT_EQ(edl.GetEditList().at(1).start, 30s);
+  EXPECT_EQ(edl.GetEditList().at(1).end, 32s);
+  EXPECT_EQ(edl.GetEditList().at(2).start, 37s);
+  EXPECT_EQ(edl.GetEditList().at(2).end, 50s);
+  EXPECT_EQ(edl.GetEditList().at(3).start, 52s);
+  EXPECT_EQ(edl.GetEditList().at(3).end, 60s);
+  EXPECT_EQ(edl.GetEditList().at(4).start, 62s);
+  EXPECT_EQ(edl.GetEditList().at(4).end, 65100ms);
```

Comment:
65.1s , for consistency with the rest of the code block?

Reply:
Yeah it was set like that before but it leads to errors (floating point) in our Jenkins workers. Works fine on my machines though. Hence I opted to keep this one as ms